config.yaml

```diff
@@ -59,3 +59,15 @@ options:
       This configuration accepts comma-separated MAC prefixes in string format.
     default: "52:54:00,fa:16:3e,06:f1:3a,00:0d:3a,00:50:56"
     type: string
+  skip-interfaces:
```

Comment:
I am curious that do we compare the pros and cons between blacklist and whitelist solution? We try to filter many special interface types here. But we may only need to filter "en[a-f0-9]*" ?

Reply:
I think we'd face similar issue with whitelist. Of the top of my head there could be
* `eno\d*`
* `ens\d*`
* `enp\ds\d*`
* `eth\d*`
And I'm sure there are more that I'm not aware of.

Reply:
I think 
* en[a-f0-9]*
* eth[a-f0-9]*
covers 90% cases in the servers.
https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/
https://github.com/systemd/systemd/blob/main/src/udev/udev-builtin-net_id.c#L1099

This is the standard naming rule that we can predecit. But there must be more and more special "virtual" interfaces. It sounds that it's easier to list the interfaces from positive side (whitelist) rather then from negative side (blacklist). Here is my opinion that may be easlier to maintain in the future.

Reply:
You are right, the whitelist approach is going to be easier than blacklist. I'll be closing this issue in favor of #33